modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectActionLocalServiceTest.java

```diff
@@ -778,6 +779,49 @@ public void testAddObjectAction() throws Exception {
 		_objectActionLocalService.deleteObjectAction(systemObjectAction);
 	}
 
+	@Test
+	public void testAddObjectActionWithCircularReference() throws Exception {
+		_testAddObjectActionWithCircularReference(2);
+
+		Class<ObjectActionThreadLocal> clazz = ObjectActionThreadLocal.class;
+
+		String originalFieldName = "_clearObjectEntryIdsMapThreadLocal";
+
+		Object originalField = ReflectionTestUtil.getFieldValue(
```

Comment:
@devpedrotavares please rename it to `originalClearObjectEntryIdsMapThreadLocal`